### PR TITLE
Wip/remove symbolic operators

### PR DESCRIPTION
**Pull request by muuki88**

### CODE REVIEW

**Comment by muuki88** [comment]
@eed3si9n / @dwijnand  could one of you have a short look if the migration looks fine so far?

The only thing I stumbled upon was  the `??` operator. You have an insights on this one?

File: src/main/scala/com/typesafe/sbt/packager/archetypes/scripts/BashStartScriptPlugin.scala [hunk 1]
```diff
@@ -55,8 +55,8 @@ object BashStartScriptPlugin extends AutoPlugin {
     bashScriptDefines ++= bashScriptExtraDefines.value,
     bashScriptReplacements := generateScriptReplacements(bashScriptDefines.value),
     // Create a bashConfigLocation if options are set in build.sbt
-    bashScriptConfigLocation <<= bashScriptConfigLocation ?? Some(appIniLocation),
-    bashScriptEnvConfigLocation <<= bashScriptEnvConfigLocation ?? None,
+    bashScriptConfigLocation := (bashScriptConfigLocation ?? Some(appIniLocation)).value,
```
Comment (muuki88) [inline src/main/scala/com/typesafe/sbt/packager/archetypes/scripts/BashStartScriptPlugin.scala]: I can't replace the `??` by simply using

```scala
bashScriptConfigLocation := Some(appIniLocation)
```

Comment (eed3si9n) [inline src/main/scala/com/typesafe/sbt/packager/archetypes/scripts/BashStartScriptPlugin.scala]: What's the warning on that?

**Comment by eed3si9n** [comment]
I only glanced at the changes, but they look ok. The tricky bits that the compiler might not catch is input tasks with `Unit`, but you don't have those keys except for in the scripted tests.

File: src/main/scala/com/typesafe/sbt/packager/archetypes/scripts/BashStartScriptPlugin.scala (hunk 1 above)
Comment (muuki88) [inline src/main/scala/com/typesafe/sbt/packager/archetypes/scripts/BashStartScriptPlugin.scala]: I haven't seen any and haven't found anything in the migration guide.
However I haven't seen this operator anywhere else yet, so I was a bit curious :)

**Comment by muuki88** [comment]
Thanks @eed3si9n for the fast reaction.

**Pull request merged**